nss-day-cohort-44/nutshell-whistling-wraiths
# Mc json setup

# Description
Added the ERD to the README
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
git fetch --all
git checkout mc-jsonSetup
check that the read me has the ERD on it
# Checklist:
- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ x] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
